tools/jbrowse2/jbrowse2.xml

```diff
@@ -0,0 +1,873 @@
+                                <cram_index>${dataset.metadata.cram_index}</cram_index>
+                            #end for
+                        </cram>
+                    #else if str($track.data_format.data_format_select) == "blastxml":
+                        <blast>
+                          #if str($track.data_format.blast_parent) != "":
+                            <parent>${track.data_format.blast_parent}</parent>
+                          #end if
+                            <protein>${track.data_format.is_protein}</protein>
+                            <min_gap>${track.data_format.min_gap}</min_gap>
+                        </blast>
+                    #else if str($track.data_format.data_format_select) == "gff":
+                        <gff>
+                          #if $track.data_format.match_part.match_part_select == "true":
+                            <match>${track.data_format.match_part.name}</match>
+                          #end if
+                        </gff>
+                    #else if str($track.data_format.data_format_select) == "paf":
+                        <paf>
+                            <genome>
+                            #for refgenome in $track.data_format.pafuseuri.refgenomes:
+                                $refgenome.annotation,
+                            #end for
+                            </genome>
+                            <genome_label>
+                            #for refgenome in $track.data_format.pafuseuri.refgenomes:
+                                $refgenome.annoname,
+                            #end for
+                            </genome_label>
+                        </paf>
+                    #else if str($track.data_format.data_format_select) == "hic":
+                        <hic>
+                        </hic>
+                    #else if str($track.data_format.data_format_select) == "cool":
+                        <cool>
+                        </cool>
+                    #else if str($track.data_format.data_format_select) == "bed":
+                        <bed>
+                        </bed>
+                    #else if str($track.data_format.data_format_select) == "sparql":
+                        <label>${track.data_format.label}</label>
+                        <sparql>
+                            <url>${track.data_format.url}</url>
+                            <query>${track.data_format.query}</query>
+                            <query_refnames>${track.data_format.query_refnames}</query_refnames>
+                        </sparql>
+                    #end if
+                    </options>
+                    #end if
+                  </track>
+            #end for
+        #end for
+    </tracks>
+    </assembly>
+    #end for
+#end if
+</root>
+]]></configfile>
+    </configfiles>
+    <inputs>
+        <repeat name="assemblies" min="1" title="Genome Assembly" help="When too many genomes are never enough">
```

Comment:
```suggestion
        <repeat name="assemblies" min="1" title="Genome Assembly" help="Visualise multiple Genomes at the same time.">
```

Reply:
`Cranky old man in the middle of unpleasant fussing comments` should definitely be adjusted.